### PR TITLE
[ALS-6893] Remove stepper and add multiple snp filters feature

### PR DESCRIPTION
**Pull request by srpiatt**
- Refactor GeneFilter store into SNP and Gene stores.
- Refactor gene filter tests into separate gene and snp test files.
- Strip infoColumns api request and use static values instead.

### CODE REVIEW

File: src/lib/components/explorer/gemone-filter/GeneSearch.svelte
Comment (JamesPeck) [inline src/lib/components/explorer/gemone-filter/GeneSearch.svelte:1]: `gemone`?

Comment (JamesPeck) [reply src/lib/components/explorer/gemone-filter/GeneSearch.svelte:1]: Wait, didn't you call this out in slack once?

Comment (srpiatt) [reply src/lib/components/explorer/gemone-filter/GeneSearch.svelte:1]: Oh... I didn't realize I had done it there too. :X I had a different filename that was gmone. T_T

**Comment by srpiatt** [comment]
Reviewed in office hours, Jul 29, 2024

**Pull request merged**